src/sentry/workflow_engine/migration_helpers/issue_alert_conditions.py

```diff
@@ -317,3 +317,49 @@ def create_percent_sessions_data_condition(
         count_type=Condition.PERCENT_SESSIONS_COUNT,
         percent_type=Condition.PERCENT_SESSIONS_PERCENT,
     )
+
+
+def create_event_unique_user_frequency_condition_with_conditions(
+    data: dict[str, Any], dcg: DataConditionGroup, conditions: list[dict[str, Any]] | None = None
+) -> DataCondition:
+    comparison_type = data.get("comparisonType", ComparisonType.COUNT)
+
+    comparison = {
+        "interval": data["interval"],
+        "value": data["value"],
+    }
+
+    if comparison_type == ComparisonType.COUNT:
+        type = Condition.EVENT_UNIQUE_USER_FREQUENCY_COUNT
+    else:
+        type = Condition.EVENT_UNIQUE_USER_FREQUENCY_PERCENT
+        comparison["comparison_interval"] = data["comparisonInterval"]
+
+    comparison_filters = []
+
+    if conditions:
+        for condition in conditions:
+            if condition["id"] in (EventAttributeFilter.id, TaggedEventFilter.id):
+                comparison_filter = {
+                    "match": condition["match"],
+                    "key": (
+                        condition["attribute"]
+                        if condition["id"] == EventAttributeFilter.id
+                        else condition["key"]
+                    ),
+                }
+                if comparison_filter["match"] not in {MatchType.IS_SET, MatchType.NOT_SET}:
+                    comparison_filter["value"] = condition["value"]
```

Comment:
both `EventAttributeFilter` and `TaggedEventFilter` are valid nested conditions here, but they will both be converted into the same Snuba condition using tags e.g. `("tags[region]", "=", "US")`

the original condition only accepts `TaggedEventFilter` but people have been using `EventAttributeFilter` in their issue alerts, so i thought to include it here. `EventAttributeFilter` is ignored by the existing condition today
https://github.com/getsentry/sentry/blob/9723427b6b31d599f4a9fcdd2351a22f11833d1a/src/sentry/rules/conditions/event_frequency.py#L728-L729